wordpress-mobile/AztecEditor-Android
# Add VideoPress support

This PR adds support for VideoPress short codes. Fixes #469

Since VideoPress videos don't have high usage in the app (it's actually a rarities) i've decided to not introduce a new shortcode class, instead playing around the already available VideoShortcodePlugin code. 
This PR just adds 2 hidden attributes (removed when switching to HTML or posting) and use those attributes to ask for video's thumb, update the video placeholder when the App returns back with the actual poster image, and use the original src when a video is tapped.

### Test
1. Open the demo app
2. See videopress videos at the bottom
They should get updated once the app is started.
3. switch to HTML mode and check that there aren't unexpected attributes added to them 

I've already a first working demo branch of wp-android. Need to fix some issue with uploading a new video.

### Review
@0nko
